src/eradius_client.erl

```diff
@@ -419,3 +422,66 @@ make_metrics_info(Options, {ServerIP, ServerPort}) ->
     ClientAddrInfo = eradius_lib:make_addr_info({ClientName, {ParsedClientIP, undefined}}),
     ServerAddrInfo = eradius_lib:make_addr_info({ServerName, {ServerIP, ServerPort}}),
     {ClientAddrInfo, ServerAddrInfo}.
+
+inc_request_counter_accounting(MetricsInfo, #radius_request{attrs = Attrs}) ->
+    Requests = ets:match_spec_run(Attrs, client_request_counter_account_match_spec_compile()),
+    [eradius_counter:inc_counter(Type, MetricsInfo) || Type <- [accountRequests | Requests]],
+    ok;
+inc_request_counter_accounting(_, _) ->
+    ok.
+
+inc_responses_counter_accounting(MetricsInfo, #radius_request{attrs = Attrs}) ->
+    Responses = ets:match_spec_run(Attrs, client_response_counter_account_match_spec_compile()),
+    [eradius_counter:inc_counter(Type, MetricsInfo) || Type <- [accountResponses | Responses]],
+    ok;
+inc_responses_counter_accounting(_, _) ->
+    ok.
+
+
+%% check if we can use persistent_term for config
+%% persistent term was added in OTP 21.2 but we can't
+%% check minor versions with macros so we're stuck waiting
+%% for OTP 22
+-ifdef(HAVE_PERSISTENT_TERM).
```

Comment:
What happens if HAVE_PERSISTENT_TERM is not defined?
No client_request_counter_account_match_spec_compile/0 at all?

Reply:
Will be used `client_request_counter_account_match_spec_compile/0` after `else` block without using `persistent_term`. Only for Erlang 21.